src/AppInstallerCLIE2ETests/Helpers/TestCommon.cs

```diff
@@ -162,95 +154,6 @@ public static RunCommandResult RunAICLICommandViaDirectProcess(string command, s
             return result;
         }
 
-        /// <summary>
-        /// This method is used when the test is run in an OS that does not support AppExecutionAlias. E,g, our build machine.
-        /// There is not any existing API that'll activate a packaged app and wait for result, and not possible to capture the stdIn and stdOut.
-        /// This method tries to call Invoke-CommandInDesktopPackage PS command to make test executable run in packaged context.
-        /// Since Invoke-CommandInDesktopPackage just launches the executable and return, we use cmd pipe to get execution results.
-        /// The final constructed command will look like:
-        ///   Invoke-CommandInDesktopPackage ...... -Command cmd.exe -Args '-c [cmd command]'
-        ///   where [cmd command] will look like: "echo stdIn | appinst.exe args > stdout.txt 2> stderr.txt &amp;amp; echo %ERRORLEVEL% > exitcode.txt"
-        /// Then this method will read the piped result and return as RunCommandResult.
-        /// </summary>
-        /// <param name="command">Command to run.</param>
-        /// <param name="parameters">Parameters.</param>
-        /// <param name="stdIn">Optional std in.</param>
-        /// <param name="timeOut">Optional timeout.</param>
-        /// <param name="throwOnTimeout">Throw on timeout.</param>
-        /// <returns>The result of the command.</returns>
-        public static RunCommandResult RunAICLICommandViaInvokeCommandInDesktopPackage(string command, string parameters, string stdIn = null, int timeOut = 60000, bool throwOnTimeout = true)

```

Comment:
There are 2 other tests directly invoking this command, we should change those 2 tests with DirectProcess too.
RegisterApplicationTest
RegisterApplicationTest_Force